iosHackthonApp/Modules/Features/MealList/Sources/MealList/MealListView.swift

```diff
@@ -14,58 +14,64 @@ public struct MealListView: View {
-                                                .foregroundColor(.white)
-                                        }))
-                .alert(isPresented: $viewModel.showingAlert) {
-                    switch viewModel.activeAlert {
-                    case .unavailable:
-                        return  Alert(
-                            title: Text(Strings.Common.sorry),
-                            message: Text(Strings.MealListScreen.UnavailableAlert.message),
-                            dismissButton: .default(Text(Strings.Common.ok)))
-                    case .collection:
-                        return Alert(
-                            title: Text(viewModel.code),
-                            message: Text(Strings.MealListScreen.CollectionAlert.message),
-                            dismissButton: .default(Text(Strings.Common.ok)))
-                    case .error:
-                        return Alert(
-                            title: Text(Strings.Common.sorry),
-                            message: Text(Strings.Common.ErrorAlert.message),
-                            dismissButton: .default(Text(Strings.Common.ok)))
+                    .navigationBarTitle(Strings.MealListScreen.title)
+                    .navigationBarItems(trailing:
+                                            Button(action: {
+                                                self.viewModel.loadMeals(forceReload: true)
+                                            }, label: {
+                                                Image(systemName: Strings.MealListScreen.Images.reload)
+                                                    .foregroundColor(.white)
+                                            }))
+                    .alert(isPresented: $viewModel.showingAlert) {
+                        switch viewModel.activeAlert {
+                        case .unavailable:
+                            return  Alert(
+                                title: Text(Strings.Common.sorry),
+                                message: Text(Strings.MealListScreen.UnavailableAlert.message),
+                                dismissButton: .default(Text(Strings.Common.ok)))
+                        case .collection:
+                            return Alert(
+                                title: Text(viewModel.code),
+                                message: Text(Strings.MealListScreen.CollectionAlert.message),
+                                dismissButton: .default(Text(Strings.Common.ok)))
+                        case .error:
+                            return Alert(
+                                title: Text(Strings.Common.sorry),
+                                message: Text(Strings.Common.ErrorAlert.message),
+                                dismissButton: .default(Text(Strings.Common.ok)))
+                        }
                     }
+                } else {
+                    Text("No meals found") // TODO: Do something nicer than this
```

Comment:
Just a temporary thing so we can differentiate easier between no meals and problems with the API. We should make a nicer ui in the future for this scenario probably